episodes/filetypes-and-metadata.md

```diff
@@ -0,0 +1,566 @@
+pop-up menu that appears, select 'napari-aicsimageio'. Note this can take a 
+while to open, so give it some time! Alternatively, you can select in Napari's 
+top menu-bar:  
+`File > Open File(s)...`
+
+![](fig/plate1-czi-napari.png){alt="A screenshot of yeast sample data shown in 
+Napari"}
+
+This image is part of a [published dataset on Image Data Resource
+](https://idr.openmicroscopy.org/search/?query=Name:idr0011-ledesmafernandez-dad4/screenD) 
+(accession number idr0011), and comes from the [OME sample data
+](https://downloads.openmicroscopy.org/images/Zeiss-CZI/idr0011/). It is a 3D 
+fluorescence microscopy image of yeast with three channels (we'll explore what
+these channels represent in the [Exploring metadata exercise
+](#exploring-metadata)). Napari has automatically recognised these three 
+channels and split them into three separate image layers. Recall that we looked
+at how Napari handles channels in the [multi-dimensional images 
+episode](multi-dimensional-images.md#channels).
+
+We can browse its metadata by selecting in the top menu-bar:  
+`Plugins > OME Tree Widget (ome-types)`
+
+![](fig/ome-tree-widget.png){alt="A screenshot of napari-aicsimageio's OME Tree 
+Widget"}
+
+This opens a panel on the right-hand side of Napari listing different metadata 
+stored in the file. In this case, we can see it is split into categories like 
+`experimenters`, `images` and `instruments`... You can click on the different 
+categories to expand them and see more detail. For example, under 
+`images > Image:0`, we see useful metadata about this specific image. We can 
+see the `acquisition_date` when this image was acquired. Also, under `pixels` 
+we can see information about the `dimension_order`, the size of different 
+dimensions (`size_c`, `size_t`, `size_x`, `size_y` and `size_z`), the type and 
+bit-depth (`type`) and importantly the pixel size (`physical_size_x`, 
+`physical_size_x_unit` etc.). The pixel size is essential for making accurate
+quantitative measurements from our images, and will be discussed [in the next 
+section of this episode](#pixel-size).
+
+This metadata is a vital record of exactly how the image was acquired and what 
+it represents. As we've mentioned in previous episodes - it's important to 
+maintain this metadata as a record for the future. It's also essential to allow
+us to make quantitative measurements from our images, by understanding factors 
+like the pixel size. Converting between file formats can result in loss of 
+metadata, so it's always worthwhile keeping a copy of your image safely in its 
+original raw file format. You should take extra care to ensure that additional 
+processing steps don't overwrite this original image.
+
+
+:::::::::::::::::::::::::::::::::::::: callout
+
+### Exploring metadata in the console
+
+Note that you can also inspect some metadata via the console:
+
+```python
+viewer.layers[0].metadata
```

Comment:
The output here is quite hard to read. It's a little off topic but we can make it more readable with 
`import rich;
rich.print(viewer.layers[0].metadata)`
![image](https://github.com/HealthBioscienceIDEAS/microscopy-novice/assets/2535165/e89a12dc-0206-4914-becf-d97e899fbf7c)

Reply:
I'll add this to the metadata section - thanks!